server/unfsd@.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=UNFS3 daemon (port %i)
+
+[Service]
+# -e /abs/path/to/exports
+# -n nfs_service_port
+# -m mount_service_port
+# -t  # tcp only
+# -p  # do not use portmapper
+# -s  # single-user mode (map file ownership)
+ExecStart=unfsd -e %h/export -n %i -m %i -t -p -s
+
+[Install]
+WantedBy=default.target
```

Comment:
This means that the service will be automatically started, when somebody logs in. This is not necessary. We want the service to be started only when somebody runs unfsd subcommand of novaboot-shell.